.gitignore

```diff
@@ -1 +1,54 @@
-node_modules
\ No newline at end of file
+# Logs
```

Comment:
Ik weet niet waar je dit bestand vandaan hebt, maar dit is niet handig. Maak gewoon zelf een `.gitignore` met de dingen die daadwerkelijk 'ignored' moeten worden.

In dit bestand wordt bijvoorbeeld `yarn.lock` ge-ignored, een zeer slecht idee :-D

Reply:
Ik heb deze uit de sanity-preview plugin. Moet dat daar ook gefixed worden?

Reply:
Ahh, goeie. Jazeker. Maak jij een ticket aan daarvoor?